code/src/components/Board.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import styled from "styled-components/macro";
+
+import { Description } from "./Description";
+import { Status } from "./Status";
+import { Avatar } from "./Avatar";
+import { Direction } from "./Direction";
+import { ActionDescription } from "./ActionDescription";
+import { BackOrRestart } from "./BackOrRestart";
+import { Fireworks } from "./Fireworks";
+
+const BoardGrid = styled.div`
+  height: 80vh;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-areas:
+    "status status status"
+    "description description description"
+    ". North ."
+    "West avatar East"
+    ". South ."
+    "instructions instructions instructions"
+    "back back back";
+`;
+
+export const Board = () => {
+  const stateUsername = useSelector((state) => state.game.username);
+  const isLoading = useSelector((state) => state.ui.isLoading);
+
+  return (
+    <>
+      {!isLoading && stateUsername && (
```

Comment:
You could comment more in the text, especially in this kind of code, to make it easier for a person who isn't familiar with the code.